cx/spark/server.js

```diff
@@ -53,13 +53,13 @@ app.post('/', async function(req, res) {
 process.on('SIGTERM', () => {
   listener.close(async ()=>{
     console.log('Closing http server.');
-    await deleteWebhooks();
+    await deleteWebhooksByUrl(targetUrl);
```

Comment:
Is this function defined? I saw you have defined a new function named `deleteWebhookById`